mkdocs.yml

```diff
@@ -12,7 +12,6 @@ nav:
   - contributing.md
   - pulplift.md
   - Let's Encrypt: letsencrypt.md
-  - FIPS: fips.md
```

Comment:
I deleted this line because the file was deleted in ae47f49ef9b79cd79851d0caa245559ceabbc7ce